api/api.ts

```diff
@@ -89,6 +89,77 @@ app.post('/upload/:tenant', async (req: express.Request, res: express.Response)
     return
 })
 
+app.get('/download/:tenant/:idFile', async (req: express.Request, res: express.Response) => {
+    const driverHeader = req.headers['driver'] as string
+    if (!driverHeader) {
```

Comment:
The driver should not be under the request. Why to fetch a file I would need anything besides its URL and my credentials?